draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1059,25 +994,18 @@ target="sec-ta"/> for guidance on how to set Ta and the STUN
 retransmit timer, RTO.
 </t>
 
-<t>The agent will receive a Binding or Allocate response. A successful
-Allocate response will provide the agent with a server reflexive
-candidate (obtained from the mapped address) and a relayed candidate
-in the XOR-RELAYED-ADDRESS attribute. If the Allocate request is
-rejected because the server lacks resources to fulfill it, the agent
-SHOULD instead send a Binding request to obtain a server reflexive
-candidate. A Binding response will provide the agent with only a
-server reflexive candidate (also obtained from the mapped
-address). The base of the server reflexive candidate is the host
```

Comment:
I think we should keep the base definitions both for server reflexive and relayed.